### PR TITLE
rename drop to drop/zero slice from

### PR DESCRIPTION
**Pull request by TheSecondComing123**

### CODE REVIEW

File: shared/src/vyxal/Elements.scala [hunk 1]
```diff
@@ -1071,8 +1071,8 @@ object Elements:
     addPart(
       Dyad,
       "İ",
-      "Drop | Collect While Unique | Complex Number",
-      List("drop", "collect-while-unique", "complex"),
+      "Zero Slice From | Collect While Unique | Complex Number",
+      List("zero-slice-from", "slice-from", "from", "drop-from", "collect-while-unique", "complex"),
```
Comment (ysthakur) [inline shared/src/vyxal/Elements.scala]:
```suggestion
      "Drop | Collect While Unique | Complex Number",
      List("zero-slice-from", "slice-from", "drop", "collect-while-unique", "complex"),
```
The `zero-slice-from` and `slice-from` keywords are a good idea, but `drop` is also a common name for this operation (also, I've never seen anyone call it `drop-from`)

**Review by ysthakur — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a tiny change (you can use the "Commit suggestion" button to apply it)

File: shared/src/vyxal/Elements.scala (hunk 1 above)
Comment (TheSecondComing123) [reply shared/src/vyxal/Elements.scala]: Personally, I've never heard of `drop`. I like the keywords, but maybe we could call it "Zero Slice From/Drop"?

Comment (RubenVerg) [reply shared/src/vyxal/Elements.scala]: Drop is definitely the most common name, for example [Scala](https://www.scala-lang.org/api/current/scala/collection/IterableOnceOps.html#drop-aba), [Haskell](https://hackage.haskell.org/package/base-4.19.0.0/docs/Prelude.html#v:drop), and [APL](https://apl.wiki/Drop) all call it "drop".

Comment (TheSecondComing123) [reply shared/src/vyxal/Elements.scala]: @RubenVerg Alright, I've changed it to Drop/Zero Slice From.

**Review by ysthakur — APPROVED** [review]
Calling it just Drop would've been too but I guess adding `/Zero Slice From` doesn't hurt. Nice job!

**Comment by TheSecondComing123** [comment]
Wait, there's still a problem. Some of the keywords aren't implemented in literate mode (e. g. `zero-slice-from` turns into `zero-slice-|`.

**Comment by ysthakur** [comment]
What error are you getting? I tried locally and it's lexed correctly

**Comment by TheSecondComing123** [comment]
@ysthakur image

It's not exactly erroring, but the output isn't correct either.

**Comment by TheSecondComing123** [comment]
@ysthakur  Oh woah it suddenly worked I must have tested it wrong.

**Comment by ysthakur** [comment]
Awesome, good to hear

**Pull request merged**